packages/kit/src/exports/public.d.ts

```diff
@@ -236,7 +236,8 @@ export interface Cookies {
 	set(
 		name: string,
 		value: string,
-		opts: import('cookie').CookieSerializeOptions & { path: string }
+		opts: import('../runtime/server/page/types.d.ts').CustomCookieSerializeOptions
```

Comment:
I'm not an expert at the types setup, but there's nothing else in this file importing from `../runtime` which makes me think it's the wrong place to put it. everything else is being imported from `private`

Also, let's drop `Custom` from the name. All of the types we defined are specific to SvelteKit. No need to prefix all of them with `Custom`...